code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Medicine.dm

```diff
@@ -1679,3 +1679,24 @@
 /datum/reagent/rmt/overdose(var/mob/living/carbon/H, var/alien)
 	if(prob(2))
 		to_chat(H, SPAN_WARNING(pick("Your muscles are stinging a bit.", "Your muscles ache.")))
+
+/datum/reagent/coagzolug
+	name = "Coagzolug"
+	description = "A medicine that was stumbled upon by accident, coagzolug encourages blood to clot and slow down bleeding. An overdose causes dangerous blood clots capable of harming the heart."
+	reagent_state = LIQUID
+	scannable = TRUE
+	color = "#bd5eb5"
+	overdose = 10
+	metabolism = REM / 3.33
+	taste_description = "throat-clenching sourness"
+	fallback_specific_heat = 1
+
+/datum/reagent/coagzolug/affect_blood(mob/living/carbon/M, alien, removed)
+	. = ..()
+	M.add_chemical_effect(CE_BLOODCLOT)
+	M.make_dizzy(5)
+
+/datum/reagent/coagzolug/overdose(var/mob/living/carbon/H, var/alien)
+	if(prob(2))
+		to_chat(H, SPAN_WARNING(pick("You feel a clot shoot through your heart!", "Your veins feel like they're being shredded!")))
+	H.add_chemical_effect(CE_CARDIOTOXIC, 1)
```

Comment:
imo add pain and do direct heart damage here

Reply:
Yeah, cardiotoxic/etc. chem effects were intended primarily for damage over time, i.e. poisons, *not* for sudden cardiac death. I'd compare it to getting stabbed in an organ versus having an organ disease; this is clearly closer to the former, so it should deal direct damage.